core/crypto/nike/hybrid/not_ctidh.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/katzenpost/katzenpost/core/crypto/rand"
 )
 
-var CTIDHX25519 nike.Scheme = &scheme{
+var CTIDH1024X25519 nike.Scheme = &scheme{
```

Comment:
What is the reason for calling this `CTIDH1024X255129` and `CTIDH-X25519` if it doesn't use CTIDH at all?
It would be good with a comment in here explaining what the purpose of this file is.

Reply:
that file only makes sense if you read the build tag at the top and look at the other files in the same dir and look at their build tags as well.

Reply:
I will remove this since it is not needed... but if you would please next time look at the build tags at the top of the source file you'll see that what's going on here is that we have a hybrid type that changes depending on if ctidh build tag is used or not. this lets us reference this type in the nikes/schemes and kem/schemes modules where we have a lookup by name helper function.